### PR TITLE
New Custom Theme for PyDy

### PR DESCRIPTION
**Pull request by sahilshekhawat**
Tarun and I created a new theme for the website. There was no way to just change the layout.html and get an entirely different website structure. Thus, we had to create a new custom theme.
For reference http://sahilshekhawat.me/pydytheme/

Master Checklist
- [x] Please explain in more detail what you have done in this thread? I'd like to understand how you implemented this? Does the code still use and support sphinx boostrap and ablog? Did you simply override the layout.html from those packages? How is the implemented?
- [x] Please remove my and Dales names from the header. This is not a personal project and we attribute the package to all of the authors.
- [x] You've committed 38 files with 20 thousand lines. Why is all of that? Can we load the common js modules from CDNs instead? If not, any js modules that we don't plan to edit should be committed in minimized form (or they should be minimized in the build process).
- [x] It seems like you swiped the code directly from quant-econ. What is the license for that code? Also, what are the licenses for all of the code you've added? We only want to add BSD/MIT compatible licensed code to the repo or we must write it ourselves.
- [x] Is this a proper sphinx theme? i.e. when we add pages via sphinx will the header be updated automatically, for example?
- [x] Where did the blog side bars go? I think we will need some side bars.
- [x] We should probably pick some better top level links to go in the header bar. Maybe: documentation, contributing, download, examples would be good ones to start with.
- [x] The "site overview" and the "table of contents" show the same information. Is this necessary?
- [x] We should only display the anchor symbol, ¶, if the mouse hovers over it.
- [x] In the Download sidbard, change the title to "download" and make sure that the dev version points to the master.zip file instead of the repo link.
- [x] Let's get rid of the "0.1.0". This is a sphinx variable that is used for software docs but is not needed here.
- [x] The logo is mighty small. I suggest making the logo as wide as the sidebar and moving the word "PyDy" to the begining or end of the menu bar. Maybe in place of "Home".
- [x] The download menu should take you to a page that explains the various download options. We need to create that page. Either create the page and fix the link or remove the "Download" menu option for now and we will deal with it in a new PR.
- [x] The "Documentation" menu item should link to "pydy.readthedocs.org".
- [x] The "Examples" menu item leads to a 404. Either create the examples page or remove this item for now and deal with it in a future PR.
- [x] The Contributing menu item should go to a page that explains how to contribute. I don't think we have that yet, so leave this for a future pr.
- [x] I was hoping we'd merge PR #12 and then you make use of that in the theme. We don't need to double commit it. I'm merging that PR now so that you can update your branch.
- [x] Is it hard to move the "Blog" section to a sidebar? That would be nice.
- [x] When I click the first three links in the "Table of Contents" the vertical space is increased below the link I clicked.
- [x] Nice work on the responsive design. Things look good to me there.
- [x] _static/logo.jpg needs to be purged from this PR.
- [x] _theme/pydy/_static/logo.jpg needs to be purged from this PR.
- [x] _theme/pydy/_static/logo.png needs to be purged from this PR.
- [x] All of the bad file endings need to be fixed.
- [x] If were are not longer using sphinx-boostrap we need to purge that from the conf.py and the readme.
- [x] The "Download" link in the main bar goes to a wiki page. Wouldn't it be better if this went to a page in the website? I'm not sure why you would like that info on the wiki.
- [x] The logo can be removed from the index.rst page since it is on the side bar.
- [x] I still don't understand why the "Examples" menu bar link doesn't have a relative link that works.
- [x] All of the `_<directories>` belong in the `source` directory alongside the `conf.py`.
- [x] Where are the blog sidebars? Does this work with ablog?

### CODE REVIEW

**Comment by moorepants** [comment]
@sahilshekhawat and @tarzzz thanks for this! It's looking pretty good. Here are some comments:
- [x] Please explain in more detail what you have done in this thread? I'd like to understand how you implemented this? Does the code still use and support sphinx boostrap and ablog? Did you simply override the layout.html from those packages? How is the implemented?
- [x] Please remove my and Dales names from the header. This is not a personal project and we attribute the package to all of the authors.
- [x] You've committed 38 files with 20 thousand lines. Why is all of that? Can we load the common js modules from CDNs instead? If not, any js modules that we don't plan to edit should be committed in minimized form (or they should be minimized in the build process).
- [x] It seems like you swiped the code directly from quant-econ. What is the license for that code? Also, what are the licenses for all of the code you've added? We only want to add BSD/MIT compatible licensed code to the repo or we must write it ourselves.
- [x] Is this a proper sphinx theme? i.e. when we add pages via sphinx will the header be updated automatically, for example?
- [ ] Where did the blog side bars go? I think we will need some side bars.
- [x] We should probably pick some better top level links to go in the header bar. Maybe: documentation, contributing, download, examples would be good ones to start with.

**Comment by moorepants** [comment]
One more:

We have a new logo, in progress: see #1, and need to incorporate that into the header or something.

**Comment by moorepants** [comment]
Also should #10 be closed?

**Comment by sahilshekhawat** [comment]
>    Please explain in more detail what you have done in this thread? I'd like to understand how you implemented this? Does the code still use and support sphinx boostrap and ablog? Did you simply override the layout.html from those packages? How is the implemented?

I have create a new theme named `pydy` in `_theme/pydy`. No, it does not uses sphinx-bootstrap but is based on the same implementation. I just went through their code base and implemented it myself. Now, creating a new sphinx theme from just html should be simple enough.

>    Please remove my and Dales names from the header. This is not a personal project and we attribute the package to all of the authors.

I will certainly do that by day after tomorrow. (I have exams until then)

>    You've committed 38 files with 20 thousand lines. Why is all of that? Can we load the common js modules from CDNs instead? If not, any js modules that we don't plan to edit should be committed in minimized form (or they should be minimized in the build process).

Yes, that can be removed. i will be changing a lot of things and there is no need for them anymore.

>    It seems like you swiped the code directly from quant-econ. What is the license for that code? Also, what are the licenses for all of the code you've added? We only want to add BSD/MIT compatible licensed code to the repo or we must write it ourselves.

The site states `© Copyright 2014, John Stachurski and Thomas J. Sargent.` Thus, I will edit the code and create something similar without using their code. (The code that I have used from their website is very small except js)

>    Is this a proper sphinx theme? i.e. when we add pages via sphinx will the header be updated automatically, for example?

Yes, its a proper sphinx theme and even now it is generated using `make`.

>    Where did the blog side bars go? I think we will need some side bars.

I have not implemented it yet, i have decided how its UI will look. Can you please give me some pointers about UI. As, it is all that is left.

>   We should probably pick some better top level links to go in the header bar. Maybe: documentation, contributing, download, examples would be good ones to start with.

Okay, I will hard-code them and introduce a side-bar.

**Comment by tarzzz** [comment]
I have closed [10](https://github.com/pydy/pydy.github.io.src/pull/10)
This is like an initial draft of the UI design. We need to clean it up, replacing the copyright code with ours and improve the UI (introducing sidebars etc.) @sahilshekhawat Let me know in case you have any concerns on the same, and good work btw!! :+1:

**Comment by moorepants** [comment]
FYI: We probably need to purge the Git history of any copyrighted code. So it may be better to rewrite the history of this with `git rebase -i` sooner than later.

**Comment by sahilshekhawat** [comment]
@moorepants I will rebase before pushing.
Also, I am currently working on this. For any copyright issue, I have removed all html code and am writing it from scratch, I am also making the website responsive for mobiles.

**Comment by sahilshekhawat** [comment]
I did rebase but it does not purge that html code from Git history but it merged it with previous commit. 
But I have changed all of the html now, so is it still a problem, also How can we purge that html code?

**Comment by moorepants** [comment]
Did you delete commits in the rebase? If you squash the commits that contain the code it wouldn't work. The easiest way to do this may be to start over with just the changes you want. If you git rm all the offending code, then commit, then squash the all your commits together in a rebase, doesn't it remove the code?

**Comment by sahilshekhawat** [comment]
Yes, it does :)
Please checkout http://sahilshekhawat.me/pydytheme/ for updated theme.

**Comment by moorepants** [comment]
This is looking good. Nice work.

I think the main thing would be to get the main menu in better order. We should probably autogenerate from some top level sphinx rst files or something. Most of what you have in the main menu are links to random stuff that should be else where.

These are my suggested main menu links: documentation, contributing, download, examples

**Comment by moorepants** [comment]
Also, let's not commit that giant jpg scan of the new logo. I'll push up a vectorized version of the logo asap so that you can utilize that.

File: _theme/pydy/_static/basic.css [hunk 1]
```diff
@@ -0,0 +1,537 @@
+/*
+ * basic.css
```
Comment (moorepants) [inline _theme/pydy/_static/basic.css]: Where did this file come from? Did you write it all?

File: _theme/pydy/_static/pydy.css_t [hunk 2]
```diff
@@ -0,0 +1,2 @@
+@import 'basic.css';
+@import 'main.css';
```
Comment (moorepants) [inline _theme/pydy/_static/pydy.css_t]: bad line ending

File: _theme/pydy/theme.conf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[theme]
+inherit = basic
+stylesheet = pydy.css
+pygments_style = tango
```
Comment (moorepants) [inline _theme/pydy/theme.conf]: bad line ending

**Comment by moorepants** [comment]
Can't we load jquery from a cdn instead of adding it to our repo?

File: _theme/pydy/_static/basic.css (hunk 1 above)
Comment (sahilshekhawat) [inline _theme/pydy/_static/basic.css]: I forgot to delete it. Deleted now.

File: _theme/pydy/_static/pydy.css_t (hunk 2 above)
Comment (sahilshekhawat) [inline _theme/pydy/_static/pydy.css_t]: Acknowledged. Will take care of that from now on.

File: _theme/pydy/theme.conf (hunk 3 above)
Comment (sahilshekhawat) [inline _theme/pydy/theme.conf]: acknowledged.

**Comment by sahilshekhawat** [comment]
I removed jquery all together.

**Comment by jcrist** [comment]
Looks great! What happened to the colorized version of the logo? I liked that better than the greyscale one up there now. Did we decide against that?

I agree with Jason on the importance of the main menu items. My vote is for `Features` (or `About`, just something with a brief summary), `Download`, `Documentation`, `Examples`, `Contributing`). I don't think the IRC channel is even used anymore (I didn't even know we had an IRC). Probably shouldn't be on there at all, if that's the case?

**Comment by moorepants** [comment]
The colorized version is available in PR #12. I'm not sure why Sahil used the scan, probably just as a placeholder.

**Comment by sahilshekhawat** [comment]
Now the theme is ready for review, its responsive for mobile devices and uses new colorized logo. I have changed main menu links too. Also, please point out UI changes like different font or colors or links which main menu items points too.

**Comment by moorepants** [comment]
Sahil things are looking great. I'm going to blast you with a bunch of nitpicks, be forwarned :)
- [ ] The "site overview" and the "table of contents" show the same information. Is this necessary?
- [x] We should only display the anchor symbol, ¶, if the mouse hovers over it.
- [x] In the Download sidbard, change the title to "download" and make sure that the dev version points to the master.zip file instead of the repo link.
- [x] Let's get rid of the "0.1.0". This is a sphinx variable that is used for software docs but is not needed here.
- [x] The logo is mighty small. I suggest making the logo as wide as the sidebar and moving the word "PyDy" to the begining or end of the menu bar. Maybe in place of "Home".
- [x] The download menu should take you to a page that explains the various download options. We need to create that page. Either create the page and fix the link or remove the "Download" menu option for now and we will deal with it in a new PR.
- [x] The "Documentation" menu item should link to "pydy.readthedocs.org".
- [x] The "Examples" menu item leads to a 404. Either create the examples page or remove this item for now and deal with it in a future PR.
- [x] The Contributing menu item should go to a page that explains how to contribute. I don't think we have that yet, so leave this for a future pr.
- [x] I was hoping we'd merge PR #12 and then you make use of that in the theme. We don't need to double commit it. I'm merging that PR now so that you can update your branch.
- [ ] Is it hard to move the "Blog" section to a sidebar? That would be nice.
- [ ] When I click the first three links in the "Table of Contents" the vertical space is increased below the link I clicked.
- [x] Nice work on the responsive design. Things look good to me there.
- [x] _static/logo.jpg needs to be purged from this PR.
- [x] _theme/pydy/_static/logo.jpg needs to be purged from this PR.
- [x] _theme/pydy/_static/logo.png needs to be purged from this PR.
- [x] All of the bad file endings need to be fixed.
- [x] If were are not longer using sphinx-boostrap we need to purge that from the conf.py and the readme.

File: _static/custom_style.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+.navbar-nav {
+float:right;
+}
+
```
Comment (moorepants) [inline _static/custom_style.css]: Why the extra spaces?

File: _theme/pydy/_static/main.css [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+	.splash .container {
+		width: auto;
+	}
+
+	.nav-toc {
+		margin-top: 10px;
+	}
+
+}
```
Comment (moorepants) [inline _theme/pydy/_static/main.css]: line ending

File: _theme/pydy/_static/main.css [hunk 6]
```diff
@@ -0,0 +1,343 @@
+@font-face {
```
Comment (moorepants) [inline _theme/pydy/_static/main.css]: Is this file copied from somewhere? If so, what is the license?

File: _theme/pydy/layout.html [hunk 7]
```diff
@@ -0,0 +1,100 @@
+{% extends "basic/layout.html" %}
+
+{% set theme_css_files = ['/_static/pydy.css', 'http://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Gentium+Basic'] %}
```
Comment (moorepants) [inline _theme/pydy/layout.html]: line width should be ~79 characters

File: _templates/layout.html [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{# Import the theme's layout. #}
```
Comment (moorepants) [inline _templates/layout.html]: Why are there two "layout.html"s?

File: source/conf.py [hunk 9]
```diff
@@ -37,7 +37,7 @@
 ]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ['../_templates']
```
Comment (moorepants) [inline source/conf.py]: Why does this have to change? Should the `_templates` dir be in the same dir as the conf.py?

**Comment by moorepants** [comment]
@sahilshekhawat This is awesome work. It is very close to being mergeable and then we can iterate on the content and such.

File: _templates/layout.html (hunk 8 above)
Comment (sahilshekhawat) [inline _templates/layout.html]: There is a `layout.html` which is default for sphinx. `_theme/pydy/layout.html` overrides it which is here defining our new layout whereas `_templates/layout.html` overrides/adds to layout.html of the theme.

File: _theme/pydy/_static/main.css (hunk 6 above)
Comment (sahilshekhawat) [inline _theme/pydy/_static/main.css]: No, its not copied. I have coded it from scratch.

File: source/conf.py (hunk 9 above)
Comment (sahilshekhawat) [inline source/conf.py]: because theme is not `_theme/` but `_theme/pydy` and it is relative to theme.

**Comment by sahilshekhawat** [comment]
> The "site overview" and the "table of contents" show the same information. Is this necessary?

"site overview" is actually "Table of contents"

```
Site Overview
-------------
.. toctree::
   :hidden:
   self
.. toctree::
   :titlesonly:
```

> We should only display the anchor symbol, ¶, if the mouse hovers over it.
> In the Download sidbard, change the title to "download" and make sure that the dev version points to the master.zip file instead of the repo link.
> Let's get rid of the "0.1.0". This is a sphinx variable that is used for software docs but is not needed here.

Done

> The logo is mighty small. I suggest making the logo as wide as the sidebar and moving the word "PyDy" to the begining or end of the menu bar. Maybe in place of "Home".

How about doing it the way "sympy.org" does, moving logo and name over the main menu.?

> The download menu should take you to a page that explains the various download options. We need to create that page. Either create the page and fix the link or remove the "Download" menu option for now and we will deal with it in a new PR.

I have created a new "Download" page in wiki and linked it to the main menu. Its almost like sympy's.

>  The "Documentation" menu item should link to "pydy.readthedocs.org".

Done

> The "Examples" menu item leads to a 404. Either create the examples page or remove this item for now and deal with it in a future PR.

Its because there is `pydytheme` in the url. once it is hosted on `pydy.org` it will open `examples/examples.html`

> The Contributing menu item should go to a page that explains how to contribute. I don't think we have that yet, so leave this for a future pr.

Done

> I was hoping we'd merge PR #12 and then you make use of that in the theme. We don't need to double commit it. I'm merging that PR now so that you can update your branch.

But I cannot access items from source folder!

> Is it hard to move the "Blog" section to a sidebar? That would be nice.

default global `Table of contents` of sphinx only accounts for files on level 1. Let me see into it and find some workaround.

> When I click the first three links in the "Table of Contents" the vertical space is increased below the link I clicked.

Fixed now.

> Nice work on the responsive design. Things look good to me there.
> _static/logo.jpg needs to be purged from this PR.
> _theme/pydy/_static/logo.jpg needs to be purged from this PR.
> _theme/pydy/_static/logo.png needs to be purged from this PR.
> All of the bad file endings need to be fixed.
> If were are not longer using sphinx-boostrap we need to purge that from the conf.py and the readme.

Done

File: _theme/pydy/_static/main.css (hunk 6 above)
Comment (moorepants) [inline _theme/pydy/_static/main.css]: nice work!

**Comment by sahilshekhawat** [comment]
`Site Overview` defined in index.rst is "Table of Contents" Thus, to include blog there, it must be included in Site Overview. Also, increased  vertical space below first three links is because only `titlesonly` is used in site overview which leads to empty list of examples. To fix this index.rst needs to be fixed.

File: _templates/layout.html (hunk 8 above)
Comment (moorepants) [inline _templates/layout.html]: got it

**Comment by tarzzz** [comment]
@moorepants This seems good to go.

**Comment by sahilshekhawat** [comment]
Just for reference: http://sahilshekhawat.me/pydytheme is always synced with this PR for quick reference.

**Comment by moorepants** [comment]
I list a bunch of things above and non are checked off. It'd be nice to know that all of my concerns were addressed.

File: source/conf.py (hunk 9 above)
Comment (moorepants) [inline source/conf.py]: The `_templates` directory needs to be in `source/_templates`.

**Comment by sahilshekhawat** [comment]
I did addressed all the issues, only one is remaining. which is space in toc, Please refer to my previous comments.

File: source/conf.py [hunk 10]
```diff
@@ -115,20 +115,20 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #html_theme = 'sphinxdoc'
-html_theme = 'bootstrap'
-html_theme_path = sphinx_bootstrap_theme.get_html_theme_path()
+html_theme = 'pydy'
+html_theme_path = ['../_theme']
```
Comment (moorepants) [inline source/conf.py]: This should be in `source/_theme`.

Comment (sahilshekhawat) [inline source/conf.py]: Why do we need to include _theme in source directory, will it not cause some unwanted pages when autogenerated?

**Comment by moorepants** [comment]
- [ ] The "Download" link in the main bar goes to a wiki page. Wouldn't it be better if this went to a page in the website? I'm not sure why you would like that info on the wiki.
- [ ] The logo can be removed from the index.rst page since it is on the side bar.
- [ ] I still don't understand why the "Examples" menu bar link doesn't have a relative link that works.
- [ ] All of the `_<directories>` belong in the `source` directory alongside the `conf.py`.
- [ ] Where are the blog sidebars? Does this work with ablog?

File: source/conf.py (hunk 10 above)
Comment (moorepants) [inline source/conf.py]: I don't understand what you are saying here.

**Comment by moorepants** [comment]
Sahil, can you please check off issues that you addressed in the lists that I make in the comments? It isn't apparent to me what you've addressed and what you haven't.

**Comment by sahilshekhawat** [comment]
> The download menu should take you to a page that explains the various download options. We need >to create that page. Either create the page and fix the link or remove the "Download" menu option for now and we will deal with it in a new PR.

You asked to create that page. :)

About relative link in examples, It will not work when index.html in not in the root of the site, as in sahilshekhawat.me/pydytheme Thus, I had to put absolute link.

I cannot checkoff points in your comment. I have tried.

**Comment by moorepants** [comment]
Oh really. Ah! Thanks for letting me know. I didn't know that you couldn't. Ok, thanks. I'll look through the conversation and see what has been addressed more carefully.

File: source/conf.py (hunk 10 above)
Comment (moorepants) [inline source/conf.py]: Also the logo should be loaded from `source/assets` if you aren't. Or we need to move it into `source/_static`.

**Comment by sahilshekhawat** [comment]
- [x] The "site overview" and the "table of contents" show the same information. Is this necessary?<br>

They are actually same, toc is defined in site-overview.
- [x] We should only display the anchor symbol, ¶, if the mouse hovers over it.
- [x] In the Download sidbard, change the title to "download" and make sure that the dev version points to the master.zip file instead of the repo link.
- [x]  Let's get rid of the "0.1.0". This is a sphinx variable that is used for software docs but is not needed here.
- [x] The logo is mighty small. I suggest making the logo as wide as the sidebar and moving the word "PyDy" to the begining or end of the menu bar. Maybe in place of "Home".
- [x]  The download menu should take you to a page that explains the various download options. We need to create that page. Either create the page and fix the link or remove the "Download" menu option for now and we will deal with it in a new PR.
- [x] The "Documentation" menu item should link to "pydy.readthedocs.org".
- [x]  The "Examples" menu item leads to a 404. Either create the examples page or remove this item for now and deal with it in a future PR.
- [ ] The Contributing menu item should go to a page that explains how to contribute. I don't think we have that yet, so leave this for a future pr.<br>

I left it for now.
- [x]  I was hoping we'd merge PR #12 and then you make use of that in the theme. We don't need to double commit it. I'm merging that PR now so that you can update your branch.
- [ ] Is it hard to move the "Blog" section to a sidebar? That would be nice.
- [ ] When I click the first three links in the "Table of Contents" the vertical space is increased below the link I clicked.<br>

Need to edit index.rst to include it.

 Nice work on the responsive design. Things look good to me there.
- [x]  _static/logo.jpg needs to be purged from this PR.
- [x]  _theme/pydy/_static/logo.jpg needs to be purged from this PR.
- [x]  _theme/pydy/_static/logo.png needs to be purged from this PR.
- [x] All of the bad file endings need to be fixed.
- [x] If were are not longer using sphinx-boostrap we need to purge that from the conf.py and the readme.

File: source/conf.py (hunk 10 above)
Comment (sahilshekhawat) [inline source/conf.py]: Acknowledged, will update.

**Comment by moorepants** [comment]
I checked off everything i could find. If you want a master checklist, like you've just pasted in at the bottom here, I'd recommend putting that in the top most comment bar.

**Comment by sahilshekhawat** [comment]
So, only one issue is remaining and the issue is that site-overview and toc are same thus, to include blog section there , it must be included  in site-overview. and also that empty space is because examples have empty list, defined in index.rst. If you still want that, let me edit index.rst.

**Comment by moorepants** [comment]
I put the master checklist at the top. You should be able to click them.

**Comment by moorepants** [comment]
You can leave the site over and toc the same for now. It isn't a big deal.

**Comment by sahilshekhawat** [comment]
Yup, I can checkoff them. Thanks :)

**Comment by sahilshekhawat** [comment]
Let me update those two things you mentioned in code. Then we are good to go?

**Comment by moorepants** [comment]
Please address **all** of the unchecked boxes in the master checklist at the top.

**Comment by moorepants** [comment]
Only check them off if there is consensus on the conclusion.

**Comment by sahilshekhawat** [comment]
@moorepants All the unmarked issues have been fixed.

**Comment by moorepants** [comment]
There are 9 unchecked boxes in the master list at the top of this PR. Can you please review those and if they have been addressed, then check them off. If not, comment on them or make the fix.

File: source/_theme/pydy/navbar.html [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+	        <div id="top">
+	            <a href="#top" title="Back to top">
+			        <i class="icon-arrow-up"></i>
+			        <span class="visuallyhidden">Back to top</span>
+			    </a>
+	        </div>
+	        <header class="header">
+				<div class="container">
+		        	<ul>
+					  <li id="pydy"><a href="/index.html">Pydy</a></li>
```
Comment (moorepants) [inline source/_theme/pydy/navbar.html:12]: This should be PyDy not Pydy.

Comment (sahilshekhawat) [inline source/_theme/pydy/navbar.html:12]: Acknowledged

**Comment by sahilshekhawat** [comment]
I have updated master check list. I replaced custom sidebar with the one currently in use in pydy.org

 You have to locally check all the changes.  Demo is not updated.

**Comment by moorepants** [comment]
ok, i'll check locally

**Comment by sahilshekhawat** [comment]
I apologize for not updating the demo,  but it's 3AM here.

**Comment by moorepants** [comment]
You should go to bed. I'm not expecting you to sit up late to finish everything.

**Comment by moorepants** [comment]
I cloned, `make clean`, `make html`, `firefox _build/html/index.html` and I get this:
pydy multibody dynamics with python pydy website - mozilla firefox_059

**Comment by moorepants** [comment]
Merging from https://github.com/pydy/pydy.github.io.src/pull/16

**Comment by moorepants** [comment]
Nice work! It is live: http://www.pydy.org/

**Comment by sahilshekhawat** [comment]
@moorepants The reason that you are seeing this is that you are simply opening html file in a browser. You must always use a server. The most simple thing is to:

```
cd _build/html
python -m SimpleHTTPServer 8000
```

and go to `http://localhost:8000`. You changed all the absolute paths to relative paths which is the reason that there is no logo in examples because its url is `http://pydy.org/examples/examples.html` and there is no `_static` folder in that folder. I am raising a new PR fixing all the bugs including responsiveness. 
Thanks for you help :)

**Pull request closed without merging**